go.mod

```diff
@@ -0,0 +1,24 @@
+module github.com/ONSdigital/dp-filter-api
+
+go 1.14
+
+require (
+	github.com/ONSdigital/dp-api-clients-go v1.9.0
+	github.com/ONSdigital/dp-graph v1.0.2
```

Comment:
I think it would be worth moving to dp-graph version v2.0.3, although this might require some changes in the models?
If we update this dependency, we will need to use the v2 path (required by go modules). I.e.:

in go.mod
```
require(
    ...
    github.com/ONSdigital/dp-graph/v2 v2.0.3
    ...
```

imports in code:
```
import (
    ...
    "github.com/ONSdigital/dp-graph/v2/...
```

Example of similar usage in: dp-dimension-importer, dp-code-list-api and dp-hierarchy-builder.

Reply:
This has now been implemented